common/datasets/librispeech.py

```diff
@@ -255,14 +266,153 @@ def get_ogg_zip_dict(create_alias_with_prefix=None):
     return ogg_zip_dict
 
 
-def export_all_datasets(path_prefix):
+def get_arpa_lm_dict(create_alias_with_path_prefix=None):
+    """
+    Download the ARPA language models from OpenSLR,
+    valid keys are: "3gram" and "4gram".
+
+    :param create_alias_with_path_prefix:
+    :return: A dictionary with Paths to the arpa lm files
+    :rtype: dict[str, Path]
+    """
+    lm_dict = {}
+
+    download_arpa_4gram_lm_job = DownloadJob(
+        url="https://www.openslr.org/resources/11/4-gram.arpa.gz",
+        target_filename="4-gram.arpa.gz",
+        checksum="f2b2d1507637ddf459d3579159f7e8099ed7d77452ff1059aeeeaea33d274613")
+    lm_dict["4gram"] = download_arpa_4gram_lm_job.out_file
+
+    download_arpa_3gram_lm_job = DownloadJob(
+        url="https://www.openslr.org/resources/11/3-gram.arpa.gz",
+        target_filename="3-gram.arpa.gz",
+        checksum="263649573475c2991d3e755eb4e690c9d2656f2b3283a1eb589e1e4e174bf874"
+    )
+    lm_dict["3gram"] = download_arpa_3gram_lm_job.out_file
+
+    if create_alias_with_path_prefix:
+        lm_prefix = os.path.join(create_alias_with_path_prefix, "LibriSpeech", "lm")
+        download_arpa_3gram_lm_job.add_alias(os.path.join(lm_prefix, "download_3gram_lm_job"))
+        download_arpa_4gram_lm_job.add_alias(os.path.join(lm_prefix, "download_4gram_lm_job"))
+    return lm_dict
+
+
+def get_static_lexicon():
     """
-    export all datasets to path_prefix/LibriSpeech/<corpus_name>.xml.gz
+    Generate the special lemmas for LibriSpeech
+
+    Librispeech uses silence, sentence begin/end and unknown, but no other special tokens.
 
-    For i6-users: physical jobs are located in: `/work/common/asr/librispeech/data/sisyphus_work_dir/`
+    :return: the lexicon with special lemmas and phonemes
+    :rtype: lexicon.Lexicon
+    """
+    lex = lexicon.Lexicon()
+    lex.add_lemma(
+        lexicon.Lemma(
+            orth=["[SILENCE]", ""],
+            phon=["[SILENCE]"],
+            synt=[""],
```

Comment:
synt should be list of list 
(here it would work technically, but should nevertheless be consistent)

Reply:
you could also leave it None as by default it will be an empty list.